test/internet/test-dns.js

```diff
@@ -390,6 +390,15 @@ TEST(function test_lookup_ipv6_explicit_object(done) {
 
 
 TEST(function test_lookup_ipv6_hint(done) {
+  // FreeBSD does not support V4MAPPED flag.
+  if (process.platform === 'freebsd') {
+    console.log(
+      '1..0 # Skipped: test_lookup_ipv6_hint is disabled on FreeBSD.'
```

Comment:
@thefourtheye if so, perhaps squeeze both `test_lookup_ipv6_hint` and `v4mapped` in there? wouldn't tap just report `internet/test-dns.js` and not the actual test name.